ksp_plugin/vessel_body.hpp

```diff
@@ -5,6 +5,13 @@
 namespace principia {
 namespace ksp_plugin {
 
+inline Vessel::Vessel(Vessel&& other)  // NOLINT(build/c++11)
```

Comment:
So this would be an `= default` if this existed, right?  Maybe a TODO to use that when VS supports it.